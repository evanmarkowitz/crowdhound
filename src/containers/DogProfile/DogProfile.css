.dog-profile-main {
  height: 90vh;
  display: flex;
  padding: 35px;
  margin-top: 60px;
}

.dog-info-wrapper {
  flex: 3;
  padding: 25px;
  display: flex;
  justify-content: space-between;
}

.dog-info-wrapper,
.user-wrapper {
  background-color: #F3F3F3;
  border: 1px solid grey;
  border-radius: 5px;
}

.dog-info-wrapper h2 {
  color: white;
  background-color: #1dbbdf;
  font-size: 65px;
  justify-self: center;
  padding: 12px;
  border-radius: 5px;
  width: 100%;
  text-align: center;
} 

.dog-image {
  width: 100%;
  height: 100%;
  flex: 1;
  margin-right: 25px;
}

.dog-info {
  flex: 1;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
}

.human-info {
  flex: 1;
}

.dog-stats {
  display: flex;
  justify-content: space-between;
}

.stat-answer {
  font-weight: bold;
}

.stat-type {
  color: grey;
}

.human-info {
  display: flex;
  justify-content: center;
  align-items: flex-end;
}

.user-wrapper {
  width: 80%;
  height: 50vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-around;
}

.user-image {
  width: 175px;
  height: 175px;
  border-radius: 50%;
  border: 3px solid #1dbbdf;
}

.chat-button {
  height: 30px;
  width: 80%;
  font-size: 20px;
}

