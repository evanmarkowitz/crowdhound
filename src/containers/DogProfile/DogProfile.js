import React from 'react'
import './DogProfile.css'
import profilePic from '../../images/dalmation.jpg';
import userImage from '../../images/bradly-cooper.jpg';


const DogProfile = () => {

  return(
    <main className='dog-profile-main'>
      <section  className = 'dog-image' 
      style={{backgroundImage: `url(${profilePic})`,
      backgroundPosition: 'center',
      backgroundSize: 'cover',
      backgroundRepeat: 'no-repeat'}}>
      </section>
      <section className='dog-info'>
          <article>
            <h2>Robert</h2>
            <div className='about-container'>
              <h3>about:</h3>
              <p className='about'>Doggo ipsum noodle horse floofs porgo shoober doggo, such treat adorable doggo vvv.  Vvv floofs tungg I am bekom fat pupperino heck wow very biscit tungg, many pats long bois dat tungg tho blep smol. Heckin angery woofer you are doin me a concern extremely cuuuuuute</p>
            </div>
          </article>
          <article className='dog-stats'>
            <div>
              <p className='stat-type'>Activity</p>
              <p className='stat-answer'>High</p>
            </div>
            <div>
              <p className='stat-type'>Breed</p>
              <p className='stat-answer'>Dalmation</p>
            </div>
            <div>
              <p className='stat-type'>Weight</p>
              <p className='stat-answer'>5 lbs</p>
            </div>
            <div>
              <p className='stat-type'>Age</p>
              <p className='stat-answer'>1</p>
            </div>
          </article>
      </section>
      <section className='human-info'>
        <article className='user-wrapper'>
          <section  className = 'user-image' 
          style={{backgroundImage: `url(${userImage})`,
          backgroundPosition: 'center',
          backgroundSize: 'cover',
          backgroundRepeat: 'no-repeat'}}>
          </section>
          <h3>Bradley Cooper</h3>
          <button className='chat-button'>Chat</button>
          <p id='zip-code'>Zip Code: <span>80211</span></p>
        </article>

      </section>

    </main>
  )
}

export default DogProfile