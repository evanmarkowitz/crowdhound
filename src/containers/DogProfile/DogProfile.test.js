import React from 'react';
import DogProfile from './DogProfile';
import { shallow } from 'enzyme';



describe('DogProfile', () => {
  let wrapper;
  beforeEach(() => {
     wrapper = shallow(<DogProfile />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})
