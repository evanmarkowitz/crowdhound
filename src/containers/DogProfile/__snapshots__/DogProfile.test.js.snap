// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DogProfile should do something 1`] = `ReactWrapper {}`;

exports[`DogProfile should map the snapshot 1`] = `ShallowWrapper {}`;

exports[`DogProfile should render dog 1`] = `ReactWrapper {}`;

exports[`DogProfile should render loading state initially 1`] = `ShallowWrapper {}`;
